apps/arweave/src/ar_nonce_limiter.erl

```diff
@@ -306,9 +306,8 @@ request_validation(H, #nonce_limiter_info{ output = Output,
 			case ar_config:use_remote_vdf_server() of
 				true ->
 					%% Wait for our VDF server(s) to validate the remaining steps.
-					spawn(fun() ->
-						timer:sleep(1000),
-						request_validation(H, Info, PrevInfo) end);
+					%% Alternatively, the network may abandon this block.
```

Comment:
We send `refuse_validation` to clear the `scheduled` flag, but if H is still the earliest unvalidated in longest chain, we would expect it to be re-validated the next time we go to apply a block, right?

Reply:
Yep